components/input/index.en-US.md

```diff
@@ -73,3 +73,7 @@ Supports all props of `Input`.
   <Input />
 </Input.Group>
 ```
+
+#### Input.Password
+
+`Input.Password` has no extra props.
```

Comment:
We should add a prop `visibilityToggle={true|false}` to hide default toggle icon.

Reply:
done